lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -817,6 +817,14 @@ galaxy:
   # your Google Analytics account.  Example: UA-XXXXXXXX-Y
   #ga_code: null
 
+  # Please enter the URL for the Plausible server (including https) so this can be used for tracking
+  # with Plausible (https://plausible.io/).
+  # plausible_server: null
+
+  # Please enter the URL for the Galaxy server so this can be used for tracking
+  # with Plausible (https://plausible.io/).
+  #plausible_domain: null
+
```

Comment:
This is missing the matomo options, can you do a `make config-rebuild` and add the changes (also the ones in `doc/source/admin/` ?

Reply:
Thanks for pointing that out! I didn't know those commands